projects/batfish/src/test/java/org/batfish/main/BatfishTest.java

```diff
@@ -176,6 +180,40 @@ public void testParseTopologyJson() throws IOException {
     assertEquals(topology.getEdges().size(), 2);
   }
 
+  @Test
+  public void testCheckValidTopology() throws IOException {
+    Map<String, Configuration> configs = new HashMap<>();
+    configs.put("h1", createConfiguration("h1", "eth0"));
+    configs.put("h2", createConfiguration("h2", "e0"));
+    //test that checking a valid topology does not throw any exception
```

Comment:
So for tests, I like to have it as unambiguous as possible what is the test. I might write the end of this function as:

```java
EdgeSet edges = new EdgeSet(Collections.singletonList(new Edge...));
Topology topology = new Topology(edges);

// Test that checkTopology does not throw
Batfish.checkTopology(configs, topology);
```

A couple of elements of this change:

* Absolutely no code after the comment other than the function under test.
* Blank line before the test to visually separate setup and test.
* Slightly simpler constructors.